### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by speedster-pta** [autobrr/autobrr#617] [Q&A] [answered]
This looks to be a really useful project. Thank you!

I'm struggling to get started on my ARM based VPS though. I've installed as per the installation instructions and get the following when confirming status

`● autobrr@ubuntu.service - autobrr service for ubuntu
     Loaded: loaded (/etc/systemd/system/autobrr@.service; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2023-01-03 21:23:14 UTC; 7min ago
   Main PID: 762137 (autobrr)
      Tasks: 10 (limit: 28701)
     Memory: 10.9M
        CPU: 188ms
     CGroup: /system.slice/system-autobrr.slice/autobrr@ubuntu.service
             └─762137 /usr/local/bin/autobrr --config=/home/ubuntu/.config/autobrr/

Jan 03 21:23:14 adguardhome-114751 autobrr[762137]: {"level":"info","time":"2023-01-03T21:23:14Z","message":"Version: 1.13.0"}
Jan 03 21:23:14 adguardhome-114751 autobrr[762137]: {"level":"info","time":"2023-01-03T21:23:14Z","message":"Commit: 5da28a0918881e7d89689eb2806fc9899fe9346a"}
Jan 03 21:23:14 adguardhome-114751 autobrr[762137]: {"level":"info","time":"2023-01-03T21:23:14Z","message":"Build date: 2022-12-31T12:38:34Z"}
Jan 03 21:23:14 adguardhome-114751 autobrr[762137]: {"level":"info","time":"2023-01-03T21:23:14Z","message":"Log-level: DEBUG"}
Jan 03 21:23:14 adguardhome-114751 autobrr[762137]: {"level":"info","time":"2023-01-03T21:23:14Z","message":"Using database: sqlite"}
Jan 03 21:23:14 adguardhome-114751 autobrr[762137]: {"level":"info","module":"server","time":"2023-01-03T21:23:14Z","message":"Starting server. Listening on 127.0.0.1:7474"}
Jan 03 21:23:14 adguardhome-114751 autobrr[762137]: {"level":"debug","module":"scheduler","time":"2023-01-03T21:23:14Z","message":"scheduler.Start"}
Jan 03 21:23:14 adguardhome-114751 autobrr[762137]: {"level":"debug","module":"indexer","time":"2023-01-03T21:23:14Z","message":"Loaded 57 indexer definitions"}
Jan 03 21:23:14 adguardhome-114751 autobrr[762137]: {"level":"info","module":"indexer","time":"2023-01-03T21:23:14Z","message":"Loaded 0 indexers"}
Jan 03 21:23:19 adguardhome-114751 autobrr[762137]: {"level":"debug","module":"scheduler","time":"2023-01-03T21:23:19Z","message":"scheduler.AddJob: job successfully added:`

I've opened port 7474 (for TCP) but when I browse to mydomain:7474 the browser says the site can't be reached / refused to connect.

I'm obviously missing something and any assistance would be greatly appreciated.

**Comment by martylukyy** [comment, marked as answer]
By default autobrr listens on `127.0.0.1` which is the recommended way when running a reverse proxy, but if you want to expose it to the internet/network then you must change the `host` in the `~/.config/autobrr/config.toml` from `127.0.0.1` to `0.0.0.0`.

Save the changes and restart autobrr with `sudo systemctl restart autobrr@USERNAME.service.`

Now that autobrr is up and running, you should be able to visit the your web UI at `http://YOUR_IP:7474` or `http://domain.ltd:7474` and proceed with your registration/login.

**Reply by speedster-pta** [reply]
Thanks! This solved my issue.

**Comment by speedster-pta** [comment]
Next question - I've got filters set up and left them overnight to test. They're working well. I've now added an action to send the torrent to qBittorrent which is giving an error as below. Testing the qBittorrent settings under client returns "ok".

`Type: QBITTORRENT
Client: qBittorrent
Filter: Doom Patrol - 264
Time: 2023-01-05 10:02:42
Rejected: error downloading torrent file for release: Doom.Patrol.S04E06.1080p.WEB.H264-CAKES: All attempts fail:
#1: unrecoverable error downloading torrent (Doom.Patrol.S04E06.1080p.WEB.H264-CAKES) file (https://torrentseeds.org/torrent/download/... ) from 'torrentseeds' - status code: 404: unrecoverable error`

**Reply by speedster-pta** [reply]
And for TL it gives this error

`Rejected: error downloading torrent file for release: Doom Patrol S04E06 1080p x265-ELiTE: All attempts fail:
#1: metainfo could not load file contents: /tmp/autobrr-3310314409: bencode: syntax error (offset: 0): unknown value type '<'
#2: metainfo could not load file contents: /tmp/autobrr-3310314409: bencode: syntax error (offset: 0): unknown value type '<'
#3: metainfo could not load file contents: /tmp/autobrr-3310314409: bencode: syntax error (offset: 0): unknown value type '<'`

**Reply by zze0s** [reply]
9 times out of 10 that's related to wrong keys being used. For TL it's RSS key, NOT PASSKEY. RSS key can be found in Profile -> Edit, scroll to bottom RSS and copy **just the random looking part of one of the urls**.

For TS potentially same issue. Check what key it is actually asking about.

That `<` message indicates it likely got a html page instead and that came because of the wrong keys used.

**Reply by s0up4200** [reply]
Your TorrentLeech error is most definitely because of a wrong RSS key.
Make sure to only add the alphanumeric part of the URL you see on your TL profile. Not the entire thing.

The same thing probably applies to your TorrentSeeds issue, because a 404 usually means the site couldn't be found, which could happen with a bad RSS key. Please double check your RID (RSS Key).
On TS: Click on your nick / Go to Settings / Security / Copy the RID (RSS Key)

**EDIT: Didnt see ze0s reply there** :)

**Reply by speedster-pta** [reply]
> 9 times out of 10 that's related to wrong keys being used. For TL it's RSS key, NOT PASSKEY. RSS key can be found in Profile -> Edit, scroll to bottom RSS and copy **just the random looking part of one of the urls**.
> 
> For TS potentially same issue. Check what key it is actually asking about.
> 
> That `<` message indicates it likely got a html page instead and that came because of the wrong keys used.

Awesome, thanks! For TL I had the full URL+key so I've just deleted the URL part. Will wait for the next filter to fire and let you know if works.

**Reply by speedster-pta** [reply]
> Your TorrentLeech error is most definitely because of a wrong RSS key. Make sure to only add the alphanumeric part of the URL you see on your TL profile. Not the entire thing.
> 
> The same thing probably applies to your TorrentSeeds issue, because a 404 usually means the site couldn't be found, which could happen with a bad RSS key. Please double check your RID (RSS Key). On TS: Click on your nick / Go to Settings / Security / Copy the RID (RSS Key)
> 
> **EDIT: Didnt see ze0s reply there** :)

Fantastic. As above I've removed the URL from the TL key. For TS I'd copied the API key and not the RSS. Whoops.

Are you perhaps able to confirm which part of the TorrentDay RSS link is the passkey?

**Reply by s0up4200** [reply]
I'm not on TorrentDay, but it should be pretty similar. There should be something in the RSS link that should be unique to you. Probably an alphanumeric string like with TL, TS and IPT.
